runbotics/runbotics-orchestrator-ui/src/src-app/components/InfoPanel/ProcessInstanceEventsDetails/ProcessInstanceEventsDetails.tsx

```diff
@@ -107,6 +111,14 @@ const ProcessInstanceEventsDetails: VFC<ProcessInstanceEventsDetailsProps> = ({
         // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [processInstanceId]);
 
+    if (
+        processId &&
```

Comment:
It is worth to move the if value to variable that name will explain what we would like to achieve in here